### DISCUSSION TITLE
Does Frepple run on Odoo.sh ?

### DISCUSSION THREAD

**Discussion by EMKATom** [frePPLe/frepple#407] [General]
... or better said, does it work together with Odoo.sh?

**Comment by jdetaeye** [comment]
Yes and no.

You can install the frepple connector on odoo.sh - it's just an odoo addon like any other. That connector addon allows frepple to pull the data it needs from odoo and push back planning results to odoo.

You can NOT install frepple itself on odoo.sh. Odoo.sh is specific for odoo, and (to my knowledge) you don't get access to install any software you want on your odoo.sh container/vm.

**Reply by EMKATom** [reply]
Johan,
thank you for your answer.
That you can not install Frepple itself on Odoo, I was already clear, I have formulated the question a little misleading.
So, I think Frepple will be a topic for us! I like very much!
Many thanks again!
